pkg/ovs/openflow/ofctrl_packetout_test.go

```diff
@@ -1275,7 +1275,7 @@ func Test_ofPacketOutBuilder_Done(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			// Specify a hardcoded seed for testing to make output predictable.
 			// #nosec G404: random number generator not used for security purposes
-			pktRand = rand.New(rand.NewSource(1))
+			pktRand = rand.New(rand.NewPCG(1, 0))
```

Comment:
I find it very unlikely that the expected test results don't need to be updated after this change. Did you run the unit tests locally?